demo/README.md

```diff
@@ -0,0 +1,48 @@
+# Demo
```

Comment:
It looks like with this PR, anyone should be able to easily download the Feldera repo and run the demos, so it would help to have even more high-level "Getting started" instructions telling people who are not familiar with our internal architecture how to actually create all required services (I guess it's `docker compose` or `start-manager.sh`?) or pointing them to appropriate docs.

Reply:
Good point, we can update the docker compose to have a profile and container entry for each demo , and then standardize how to call them.

It could be already this PR or a next one.

Reply:
Even better: add a very short section on the top-level README, right after the instructions for how to bring up the pipeline manager. E.g.:

```
### Run a demo

Once you've brought up a pipeline-manager using the above instructions, you can run one of our many demos. For example:

$: <one command for the most simple demo>

This command will bring up a program named foo and a pipeline named bar. You can view them at <urls>.

Once you've tried this, we suggest exploring the full list of our demos [here](link-to-this-folder).
```